source/_components/tahoma.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Tahoma"
+description: "Instructions on how to integrate Somfy Tahoma devices with Home Assistant."
+date: 2017-07-18 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: somfy.png
+ha_category: Hub
+ha_release: 0.41
+---
+
+
+The `Tahoma` component platform is used as an interface to the [tahomalink.com](https://www.tahomalink.com) website. It adds actually covers and the sun sensor from tahoma platform.
+
+To use your Tahoma devices in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+tahoma:
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+  exclude: [BridgeHUEComponent, HueLampHUEComponent, PodComponent]
+```
+
+Configuration variables:
```

Comment:
You can remove this header since you have implemented the configuration tag instead.